pkg/monitor/backenddisruption/disruption_backend_sampler_test.go

```diff
@@ -24,10 +24,10 @@ func TestBackendSampler_checkConnection(t *testing.T) {
 		case req.URL.Path == "/302-bad-response":
 			w.WriteHeader(302)
 			w.Write([]byte("302-bad-response"))
-		//case req.URL.Path == "/302": // the client-auto-follows
-		//	w.Header().Set("Location", "http://google.com")
-		//	w.WriteHeader(302)
-		//	w.Write([]byte("302"))
+		case req.URL.Path == "/302": // the client-auto-follows
+			w.Header().Set("Location", "http://google.com")
+			w.WriteHeader(302)
+			w.Write([]byte("302"))
```

Comment:
This looks like it was temporarily commented out for some reason and then accidentally committed.